### DISCUSSION TITLE
درگاه زرین پال

### DISCUSSION THREAD

**Discussion by hamidrezashahnazari** [Sina-Soltani/Parbad#386] [Q&A]
سلام و وقت بخیر من از درگاه زرین پال استفاده میکنم اگر ساکسس شد  مقدار authority  رو چطور به ما برمیگردونه تا بتونیم  verify کنیم

**Comment by Alisheikh4444** [comment]
سلام مقدار discount و چطوری توی درخواستمون بگونجونیم تا از کد تخفیفی که توی خود زرین پال ایجاد کردیم استفاده کنیم؟
